src/Powercord/plugins/pc-emojiUtility/index.js

```diff
@@ -22,10 +49,18 @@ module.exports = class EmojiUtility extends Plugin {
     return /^<a?:([a-zA-Z0-9_]+):([0-9]+)>$/;
   }
 
-  getGuildUrl (guildId) {
+  getEmojiUrlRegex () {
+    return /https:\/\/cdn.discordapp.com\/emojis\/(\d+)/;
+  }
+
+  getGuildRoute (guildId) {
     const selectedChannelId = getSelectedChannelState()[guildId];
 
-    return APP_URL_PREFIX + (selectedChannelId ? Routes.CHANNEL(guildId, selectedChannelId) : Routes.GUILD(guildId)); // eslint-disable-line new-cap
+    return (selectedChannelId ? Routes.CHANNEL(guildId, selectedChannelId) : Routes.GUILD(guildId)); // eslint-disable-line new-cap
```

Comment:
would prefer if this was a multiline ternary, does eslint not have rules for that set up 🤔